diff -up sdrangel-7.19.1/plugins/channelrx/demoddatv/datvideorender.cpp.omv~ sdrangel-7.19.1/plugins/channelrx/demoddatv/datvideorender.cpp
--- sdrangel-7.19.1/plugins/channelrx/demoddatv/datvideorender.cpp.omv~	2024-04-08 15:59:53.219418436 +0200
+++ sdrangel-7.19.1/plugins/channelrx/demoddatv/datvideorender.cpp	2024-04-08 16:14:48.964045644 +0200
@@ -279,7 +279,7 @@ bool DATVideoRender::preprocessStream()
     m_metaData.Width = m_videoDecoderCtx->width;
     m_metaData.Height = m_videoDecoderCtx->height;
     m_metaData.BitRate = m_videoDecoderCtx->bit_rate;
-    m_metaData.Channels = m_videoDecoderCtx->channels;
+    m_metaData.Channels = m_videoDecoderCtx->ch_layout.nb_channels;
     m_metaData.CodecDescription = QString("%1").arg(videoCodec->long_name);
     m_metaData.OK_VideoStream = true;
 
@@ -305,8 +305,8 @@ bool DATVideoRender::preprocessStream()
         //m_audioDecoderCtx = m_formatCtx->streams[m_audioStreamIndex]->codec; // old style
 
         qDebug() << "DATVideoRender::preprocessStream: audio: "
-        << " channels: " << m_audioDecoderCtx->channels
-        << " channel_layout: " << m_audioDecoderCtx->channel_layout
+        << " channels: " << m_audioDecoderCtx->ch_layout.nb_channels
+        << " channel_layout: " << m_audioDecoderCtx->ch_layout.u.mask
         << " sample_rate: " << m_audioDecoderCtx->sample_rate
         << " sample_fmt: " << m_audioDecoderCtx->sample_fmt
         << " codec_id: "<< m_audioDecoderCtx->codec_id;
@@ -625,9 +625,9 @@ void DATVideoRender::setResampler()
     }
 
     m_audioSWR = swr_alloc();
-    av_opt_set_int(m_audioSWR, "in_channel_count",  m_audioDecoderCtx->channels, 0);
+    av_opt_set_int(m_audioSWR, "in_channel_count",  m_audioDecoderCtx->ch_layout.nb_channels, 0);
     av_opt_set_int(m_audioSWR, "out_channel_count", 2, 0);
-    av_opt_set_int(m_audioSWR, "in_channel_layout",  m_audioDecoderCtx->channel_layout, 0);
+    av_opt_set_int(m_audioSWR, "in_channel_layout",  m_audioDecoderCtx->ch_layout.u.mask, 0);
     av_opt_set_int(m_audioSWR, "out_channel_layout", AV_CH_LAYOUT_STEREO, 0);
     av_opt_set_int(m_audioSWR, "in_sample_rate", m_audioDecoderCtx->sample_rate, 0);
     av_opt_set_int(m_audioSWR, "out_sample_rate", m_audioSampleRate, 0);
@@ -637,9 +637,9 @@ void DATVideoRender::setResampler()
     swr_init(m_audioSWR);
 
     qDebug() << "DATVideoRender::setResampler: "
-        << " in_channel_count: " <<  m_audioDecoderCtx->channels
+        << " in_channel_count: " <<  m_audioDecoderCtx->ch_layout.nb_channels
         << " out_channel_count: " << 2
-        << " in_channel_layout: " <<  m_audioDecoderCtx->channel_layout
+        << " in_channel_layout: " <<  m_audioDecoderCtx->ch_layout.u.mask
         << " out_channel_layout: " <<  AV_CH_LAYOUT_STEREO
         << " in_sample_rate: " << m_audioDecoderCtx->sample_rate
         << " out_sample_rate: " << m_audioSampleRate
